### PR TITLE
Add docker compose dev

### LINKED ISSUE #9: Add docker-compose.dev.yml

### PR DESCRIPTION
**Pull request by ahmadgaz**
Resolves #9

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
-*.yaml
-*.yml
+data.yaml
+data.yml
```
Comment (evanugarte) [inline .gitignore]: this should ignore all other yml files
```suggestion
*.yaml
*.yml
!docker-compose.dev.yml
!docker-compose.yml
```

File: docker-compose.yml [hunk 2]
```diff
@@ -8,8 +8,7 @@ services:
             context: .
         command:
             - --config=/server/data.yaml
-            - --host=0.0.0.0
-            - --port=8001
+            - -vvv
```
Comment (evanugarte) [inline docker-compose.yml]: we dont need any changes to this file

File: docker-compose.dev.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+version: "3.8"
```
Comment (evanugarte) [inline docker-compose.dev.yml:1]: besides this line, i think we can remove all other double quotes in this line and everything should still work

File: docker-compose.yml (hunk 2 above)
Comment (ahmadgaz) [reply docker-compose.yml]: Done

File: docker-compose.dev.yml (hunk 3 above)
Comment (ahmadgaz) [reply docker-compose.dev.yml:1]: Done

**Review by evanugarte — APPROVED** [review]

**Pull request merged**